src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -17,17 +17,12 @@ public static class ApplicationInsightsExtensions
     {
         public static IApplicationBuilder UseApplicationInsightsRequestTelemetry(this IApplicationBuilder app)
```

Comment:
Thanks, @Eilon. Since the `Use` prefix is important, the raw `app.UseMiddleware<RequestTrackingMiddleware>()` call feels more clear and descriptive than the `app.UseApplicationInsightsRequestTelemetry()` we have now. Why do we need to provide an Application Insights-specific extension method?